modules/ROOT/pages/scheduler-concept.adoc

```diff
@@ -2,72 +2,213 @@
+
+[%header,cols="1a,4a"]
 |===
-|Attribute |Description |Default
+| Attribute XML | Description
+
+|`disallowConcurrentExecution`
+| When this attribute is set to `true`, the Scheduler skips its scheduled execution if a previous execution is still running. Once the flow is done, the Scheduler executes at the next configured interval. The default value is `false`.
+|===
+
+In the following example, the Scheduler skips its execution if the flow takes longer to complete than the fixed frequency of 10 milliseconds:
+
+[source,xml]
+----
+<flow name="scheduler-disallowConcurrentExecution-ex" >
+  <scheduler doc:name="Scheduler" disallowConcurrentExecution="true" >
+    <scheduling-strategy>
+      <fixed-frequency frequency="10" timeUnit="MILLISECONDS"/>
+    </scheduling-strategy>
+  </scheduler>
+  <!-- processors here -->
+</flow>
+----
+
+Each time a Scheduler skips its execution, Mule logs an `Execution skipped` message, for example (edited for readability):
+
+[source,logs]
+----
+INFO  2022-11-09 15:15:43,082 ...
+...scheduler.DefaultSchedulerMessageSource:
+Flow 'scheduler-disallowConcurrentExecution-ex' is already running and
+'disallowConcurrentExecution' is set to 'true'. Execution skipped.
+
+INFO  2022-11-09 15:15:43,083 ...
+...scheduler.DefaultSchedulerMessageSource:
+Flow 'scheduler-disallowConcurrentExecution-ex' is already running and
+'disallowConcurrentExecution' is set to 'true'. Execution skipped.
+----
+
+[[props_fixed_frequency]]
+=== Fixed Frequency (`<fixed-frequency/>`)
+
+The fixed frequency scheduling strategy accepts the following attributes:
+
+[%header,cols="1a,1a,1a,4a"]
+|===
+|Attribute | XML | Default | Description
+
 |*Frequency*
-|The frequency at which the Scheduler triggers the flow. You select the unit of time in the *Time unit* field. Setting the value to `0` or a negative value causes the default value to be used.
-|
-1000
+| `frequency`
+| `1000`
+|The frequency at which the Scheduler triggers the flow given the `timeUnit` setting.
 
 | *Start Delay*
-| The amount of time to wait before triggering the flow for the first time after the application is started. This value is expressed in the same unit of time as the frequency.
-| 0
+| `startDelay`
+| `0`
+| The amount of time to delay execution of the Scheduler for the first time after the application starts. This property uses the same `timeUnit` as the frequency.
 
 | *Time Unit*
-| The time unit for the values of *Frequency* and *Start Delay*. The possible values are:
+| `timeUnit`
+| `MILLISECONDS`
+| One of the following time units for the values of `frequency` and `startDelay`:
+
+`MILLISECONDS`, `SECONDS`, `MINUTES`, `HOURS`, or `DAYS`
+
+|===
+
+This strategy delays execution of the Scheduler for 10 seconds and then executes every 10 seconds:
```

Comment:
It is not clear what executes every 10 seconds. Is it the scheduler or the flow? Can you reword this to clarify?

Reply:
done